README.md

```diff
@@ -119,7 +119,7 @@ This will produce an output file `some_events.edm4hep.json`. That has the follow
 ```
 
 As also stated in the [podio
-documentation](https://github.com/AIDASoft/podio/blob/master/doc/advanced_topics.md#dumping-json)
+documentation](https://github.com/AIDASoft/podio/blob/main/doc/advanced_topics.md#dumping-json)
```

Comment:
podio is still on master. So this one will have to be unchanged for now.

Reply:
Sorry I thought I checked there were only links to EDM4hep and replaced all